### PR TITLE
Fix the issue that some job don't require space.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

File: src/validator/AdminJobValidator.h [hunk 1]
```diff
@@ -19,7 +19,9 @@ class AdminJobValidator final : public Validator {
     AdminJobValidator(Sentence* sentence, QueryContext* context)
             : Validator(sentence, context) {
         sentence_ = static_cast<AdminJobSentence*>(sentence);
-        if (sentence_->getOp() != meta::cpp2::AdminJobOp::ADD) {
+        if (sentence_->getOp() != meta::cpp2::AdminJobOp::ADD ||
+            (sentence_->getCmd() != meta::cpp2::AdminCmd::REBUILD_TAG_INDEX ||
+              sentence_->getCmd() != meta::cpp2::AdminCmd::REBUILD_EDGE_INDEX)) {
```
Comment (bright-starry-sky) [inline src/validator/AdminJobValidator.h:24]: sentence_->getCmd() != meta::cpp2::AdminCmd::REBUILD_TAG_INDEX &&
sentence_->getCmd() != meta::cpp2::AdminCmd::REBUILD_EDGE_INDEX

Comment (Shylock-Hg) [reply src/validator/AdminJobValidator.h:24]: Good catch.

Comment (bright-starry-sky) [inline src/validator/AdminJobValidator.h:24]: I suggest extract a method to check the space require, to avoid missing new items of `AdminCmd` .

Comment (Shylock-Hg) [reply src/validator/AdminJobValidator.h]: Fixed.

Comment (Shylock-Hg) [reply src/validator/AdminJobValidator.h]: Fixed.

File: src/validator/AdminJobValidator.h [hunk 2]
```diff
@@ -29,6 +29,29 @@ class AdminJobValidator final : public Validator {
...
+                }
+                break;
+            case meta::cpp2::AdminJobOp::SHOW_All:
+            case meta::cpp2::AdminJobOp::SHOW:
+            case meta::cpp2::AdminJobOp::STOP:
+            case meta::cpp2::AdminJobOp::RECOVER:
+                return false;
+        }
+    }
+
```
Comment (bright-starry-sky) [inline src/validator/AdminJobValidator.h:54]: 👍

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Review by darionyaphet — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
Well done

**Review by jievince — APPROVED** [review]

**Pull request merged**